packages/components/Notification/src/Notification.helper.tsx

```diff
@@ -62,8 +62,8 @@ export const NotificationButton = stagedComponent((props: NotificationButtonProp
       iconColor: props.color,
       fontSize: 15,
```

Comment:
Oh I didn't think about that. I'll check

Reply:
Okay I looked into it, the values here correspond to body2strong, but I couldn't really figure out how to use a token that's meant for text in Button.customize. Here are some ways I tried it that didn't work:

<img width="250" alt="Screen Shot 2022-10-07 at 4 16 49 PM" src="https://user-images.githubusercontent.com/78454019/194674590-513c5252-388b-417c-b8c1-c0c7d9e02964.png">

<img width="281" alt="Screen Shot 2022-10-07 at 4 17 14 PM" src="https://user-images.githubusercontent.com/78454019/194674577-17ac4d80-26ed-455d-9783-ac7caa8d6883.png">

Reply:
Oh well, it's totally possible that we don't support it yet since we're still working on getting all the tokens in place. I figured it was worth asking.

Reply:
Hm, that's odd, the first of those should have worked. That should be looked into.

Reply:
I'll make a bug for it